styles.css

```diff
@@ -405,13 +417,14 @@ h2 {
   display: flex;
   flex-direction: column;
   margin-top: 15vh;
-  padding: 20px;
+  /* padding: 20px; */
```

Comment:
While you did a very wonderful job ⚡⚡, please remember to get rid of unsed code to keep your work clean